### ISSUE TITLE
Add noise data to timer tag operational tag

### ISSUE THREAD

**Issue by KevinSee** [mackerman44/telemetyr#30]
When summarizing operational time, use timer tag (575), any noise tags (975) and sentinel tag (550), not just timer tag, in case the timer tag was vandalized.

**Comment by mackerman44** [comment]
Do we really want to include observations from the noise tags? During the pilot season, I deliberately ignored noise when summarising operation times, because at times I found when noise was too bad the site actually couldn't read the timer tag (and presumably fish). The sentinel observations makes sense.

**Comment by mackerman44** [comment]
I updated summarise_timer_data so that is uses code endings 575 OR 550. I did not add noise 995s for the time being.

I suppose it comes down to the question of whether we want to summarise the time that the receivers were running OR the time that we believe receivers could detect fish. When I first did this summary I was really interested in "What proportion of the season do I believe the receivers could detect fish?" the really summary of performance.
